db/types.go

```diff
@@ -32,6 +32,15 @@ func (u *User) HasRoleFor(address string, role UserRole) bool {
 	return false
 }
 
+func (u *User) IsMemberOf(address string) bool {
```

Comment:
I think that it should be commented and moved to `user.go`

Reply:
This function is very similar to the function `HasRoleFor`: It does not use the DB just the properties of the structs. Therefore both functions need to be in the same place. If you believe this is user.go, then I will move both of them